Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM progrium/busybox
```

Comment:
Yes. It provides lib needed by the cfs binary, especially cadvisor dep. I tried to build it statically, but it is harder than I think. Current image size is 18.21MB, so it is fine IMO.